ksql-engine/src/test/java/io/confluent/ksql/services/KafkaTopicClientImplTest.java

```diff
@@ -498,6 +564,17 @@ private static DescribeTopicsResult describeTopicReturningUnknownPartitionExcept
     return describeTopicsResult;
   }
 
+  private static DescribeTopicsResult describeTopicTopicAuthorizationException(String topicName) {
+    final DescribeTopicsResult describeTopicsResult = niceMock(DescribeTopicsResult.class);
+    expect(describeTopicsResult.all())
+        .andReturn(failedFuture(
+            new TopicAuthorizationException(
+                AclErrorMessageUtil.generateErrorMessageSingleTopic(AclOperation.DESCRIBE, topicName)
```

Comment:
nit: Let's not put the specific error message here (an empty message as you've done in `createTopicAuthorizationException()` is better), per the earlier discussion about verifying where the error message is generated.